snmp/metadata.csv

```diff
@@ -1,4 +1,5 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+snmp.devices_monitored,gauge,,,,Devices monitored count. Make 'sum by {X}' queries to count all the Devices with the tag X.,0,snmp,
```

Comment:
Can we explicit what is the unique tag key ? i.e what is `foo` in the query `sum:snmp.devices_monitored{FILTERING_TAGS} by {foo}`

Also, the description is incorrect a `sum by {X}` query does not count devices with the tag X, it counts the number of different values the tag key X has.